packages/libmikmod/SPECS/libmikmod.spec

```diff
@@ -0,0 +1,249 @@
+Summary:        A MOD music file player library
+Name:           libmikmod
+Version:        3.3.11.1
+Release:        7%{?dist}
+License:        GPLv2 and LGPLv2+
+URL:            http://mikmod.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/mikmod/libmikmod-%{version}.tar.gz
+Patch0:         libmikmod-64bit.patch
+Patch1:         libmikmod-multilib.patch
+Patch2:         libmikmod-cflags.patch
+BuildRequires:  gcc
+BuildRequires:  alsa-lib-devel pulseaudio-libs-devel
```

Comment:
We don't have alsa-lib-devel or pulseaudio-libs-devel on irix - so building will fail here. How are you building this so it works? The correct build that skips checks is `rpmbuild -ba PACKAGE.spec --nocheck` - please don't skip dependency checking.